packs/tfe_fdo_nomad/README.md

```diff
@@ -0,0 +1,156 @@
+1. `NOMAD_CACERT` - Path to a PEM encoded CA cert file to use to verify the Nomad server SSL certificate.
+1. `NOMAD_CLIENT_CERT` - Path to a PEM encoded client certificate for TLS authentication to the Nomad server. Must also specify NOMAD_CLIENT_KEY.
+1. `NOMAD_CLIENT_KEY` - Path to an unencrypted PEM encoded private key matching the client certificate from NOMAD_CLIENT_CERT.
+
+After setting up the environment variables, the pack can be setup using the following steps:
+
+1. Create Namespace for TFE job and TFE agent job.
+
+   1. Run `nomad namespace apply terraform-enterprise` to create the `terraform-enterprise` namespace. This is the default namespace that is used to bring up TFE Job.
+   1. Run `nomad namespace apply tfe-agents` to create the `tfe-agents` namespace. This is the default namespace that is used to bring up TFE Agent Job.
+
+
+2. Create a Nomad ACL policy file `terraform_enterprise_policy.hcl` with the content below:
+```hcl
+  namespace "tfe-agents" {
+  capabilities = ["submit-job","dispatch-job", "list-jobs", "read-job", "read-logs" ]
+  }
+  ```
+
+3. Apply the Nomad policy using:
+  ```bash
+  $ nomad acl policy apply \
+   -namespace terraform-enterprise -job tfe-job \
+   -group tfe-group -task tfe-task \
+   terraform-enterprise-policy ./terraform_enterprise_policy.hcl
+  ``` 
+
+4. Create the necessary Nomad Variables for each job. 
+  
+  These contain sensitive data that are required like certs, licenses and passwords. 
+  Create a variable specification file: 
+
+  ```hcl
+  # spec.nv.hcl
+  path      = "nomad/jobs/tfe-job"
+  namespace = "terraform-enterprise"
+
+  items {
+    # TFE DB password. Mapped to the TFE_DB_PASSWORD environment variable.
+    db_password = ""
+
+    # The field should contain the base64 encoded value of the cert. Mappped to the TFE_TLS_CERT_FILE environment variable.
+    cert = ""
+
+    # The field should contain the base64 encoded value of the bundle. Mapped to the TFE_TLS_CA_BUNDLE_FILE environment variable.
+    bundle = ""
+
+    # The field should contain the base64 encoded value of the key. Mappped to the TFE_TLS_KEY_FILE environment variable.
+    key = ""
+  
+    # A valid TFE license. Mapped to the TFE_LICENSE environment variable.
+    tfe_license = ""
+    
+    # Object storage access key. Mapped to the TFE_OBJECT_STORAGE_S3_SECRET_ACCESS_KEY environment variable.
+    s3_secret_key = ""
+
+    # The field should contain the base64 encoded value of the Nomad CA. Mapped to the TFE_RUN_PIPELINE_NOMAD_TLS_CONFIG_CA_CERT environment variable.
+    nomad_ca_cert = ""
+
+    # The field should contain the base64 encoded value of the Nomad cert. Mapped to the TFE_RUN_PIPELINE_NOMAD_TLS_CONFIG_CLIENT_CERT environment variable.
+    nomad_cert = ""
+
+    # The field should contain the base64 encoded value of the Nomad cert's key. Mapped to the TFE_RUN_PIPELINE_NOMAD_TLS_CONFIG_CLIENT_KEY environment variable.
+    nomad_cert_key = ""
+
```

Comment:
For a future version, it would be nice to be able to get the TLS keys/certs/CA/etc. from a Vault cluster instead of having to add them to Nomad Vars

Reply:
thanks @sofixa, duly noted!